contributing/code/bc.rst

```diff
@@ -51,53 +51,28 @@ sticks to these rules.
     The exception to this rule are interfaces tagged with ``@internal``. Such
     interfaces should not be used or implemented.
 
-If you want to implement an interface, you should first make sure that the
-interface is an API interface. You can recognize API interfaces by the ``@api``
-tag in their source code::
-
-    /**
-     * HttpKernelInterface handles a Request to convert it to a Response.
-     *
-     * @author Fabien Potencier <fabien@symfony.com>
-     *
-     * @api
-     */
-    interface HttpKernelInterface
-    {
-        // ...
-    }
-
-If you implement an API interface, we promise that we won't ever break your
-code. Regular interfaces, by contrast, may be extended between minor releases,
-for example by adding a new method. Be prepared to upgrade your code manually
-if you implement a regular interface.
-
-.. note::
-
-    Even if we do changes that require manual upgrades, we limit ourselves to
-    changes that can be upgraded easily. We will always document the precise
-    upgrade instructions in the UPGRADE file in Symfony's root directory.
+If you implement an interface, we promise that we won't ever break your code.
 
 The following table explains in detail which use cases are covered by our
 backwards compatibility promise:
 
-+-----------------------------------------------+---------------+---------------+
-| Use Case                                      | Regular       | API           |
-+===============================================+===============+===============+
-| **If you...**                                 | **Then we guarantee BC...**   |
-+-----------------------------------------------+---------------+---------------+
-| Type hint against the interface               | Yes           | Yes           |
-+-----------------------------------------------+---------------+---------------+
-| Call a method                                 | Yes           | Yes           |
-+-----------------------------------------------+---------------+---------------+
-| **If you implement the interface and...**     | **Then we guarantee BC...**   |
-+-----------------------------------------------+---------------+---------------+
-| Implement a method                            | No [1]_       | Yes           |
-+-----------------------------------------------+---------------+---------------+
-| Add an argument to an implemented method      | No [1]_       | Yes           |
-+-----------------------------------------------+---------------+---------------+
-| Add a default value to an argument            | Yes           | Yes           |
-+-----------------------------------------------+---------------+---------------+
++-----------------------------------------------+-----------------------------+
+| Use Case                                      | Backwards Compatibility     |
```

Comment:
I think a table structure is very nice here. It's clear and it's consistent with the other "promises tables" below.